### PR TITLE
Add job not running to FAQ

### PR DESCRIPTION
**Pull request by Carldkennedy**
May need fleshing out with memory issues too?

### CODE REVIEW

**Comment by jkwmoore** [comment]
I think we may want to flesh this out a bit more, requesting too much time is only one reason for a job not starting. The most common _generic_ reason is submitting a request which is unfulfillable. (Assuming the cluster is still healthy.)

Unfulfillable requests can have many origins: 

- requesting too many cores in the wrong parallel environment.
- requesting too much memory in the wrong parallel environment.
- requesting too much time etc...

Probably sensible for us to rework this to first explain/summarise the generic reason this happens (unfulfillable request), examples of unfulfillable requests (and why) followed by how to correct them (qalter or resubmit.)

Best to look at the man pages for scontrol / qalter to be sure about which of the requestable resources can be adjusted after submission (and how) and which can't (resubmit.)

File: FAQs.rst [hunk 1]
```diff
@@ -96,6 +96,76 @@ Please refer to our :ref:`Choosing appropriate compute resources page <Choosing-
...
+I've submitted a job but it's not running
+-----------------------------------------
+
+I submitted a job and after several days it is still waiting in the queue. How can I resolve this?
+There are a multitude of factors which could be causing your job to queue for a long time or to not run at all.
+Occasionally parts of the system may be in a maintenance period or may be utlised to capacity.   
+A few things to consider which would cause your job to not run at all:
+
+* Did you request an acceptable amount of memory for a given node? (e.g. on Bessemer 192GB or less)
+* Did you request too much memory in the wrong parallel environment? (e.g on Bessemer OpenMP `-l rmem=16G` with 16 cores would request 16*16=256G exceeding node memory)
```
Comment (jkwmoore) [inline FAQs.rst:108]: Guess this should be ShARC.
```suggestion
* Did you request too much memory in the wrong parallel environment? (e.g on ShARC, OpenMP `-l rmem=16G` with 16 cores would request 16*16=256G exceeding node memory)
```

File: FAQs.rst [hunk 2]
```diff
@@ -96,6 +96,76 @@ Please refer to our :ref:`Choosing appropriate compute resources page <Choosing-
...
 
+I've submitted a job but it's not running
+-----------------------------------------
+
+I submitted a job and after several days it is still waiting in the queue. How can I resolve this?
+There are a multitude of factors which could be causing your job to queue for a long time or to not run at all.
+Occasionally parts of the system may be in a maintenance period or may be utlised to capacity.   
+A few things to consider which would cause your job to not run at all:
+
+* Did you request an acceptable amount of memory for a given node? (e.g. on Bessemer 192GB or less)
```
Comment (jkwmoore) [inline FAQs.rst:107]:
```suggestion
* Did you request an acceptable amount of memory for a given node? (e.g. on Bessemer, 192GB or less.)
```

File: FAQs.rst [hunk 3]
```diff
@@ -96,6 +96,76 @@ Please refer to our :ref:`Choosing appropriate compute resources page <Choosing-
...
+* Did you request too much memory in the wrong parallel environment? (e.g on Bessemer OpenMP `-l rmem=16G` with 16 cores would request 16*16=256G exceeding node memory)
+* Did you request too many cores in the wrong parallel environment?
+  )
+* Did you request too much time? (e.g on Bessemer more than 168 hrs)
```
Comment (jkwmoore) [inline FAQs.rst:111]:
```suggestion
* Did you request too much time? (e.g on Bessemer, more than 168 hrs.) 
```

File: FAQs.rst [hunk 4]
```diff
@@ -96,6 +96,75 @@ Please refer to our :ref:`Choosing appropriate compute resources page <Choosing-
...
+-----------------------------------------
+
+I submitted a job and after several days it is still waiting in the queue. How can I resolve this?
+There are a multitude of factors which could be causing your job to queue for a long time or to not run at all.
+Occasionally parts of the system may be in a maintenance period or may be utlised to capacity.   
+A few things to consider which would cause your job to not run at all:
+
+* Did you request an acceptable amount of memory for a given node? (e.g. on Bessemer, 192GB or less.)
+* Did you request too much memory in the wrong parallel environment? (e.g on ShARC, OpenMP `-l rmem=16G` with 16 cores would request 16*16=256G exceeding node memory.)
+* Did you request too many cores in the wrong parallel environment?
```
Comment (jkwmoore) [inline FAQs.rst:109]:
```suggestion
* Did you request too many cores in the wrong parallel environment? (e.g on ShARC,  `-pe openmp 40` would request 40 cores, exceeding a single node's core count.)
```

File: FAQs.rst [hunk 5]
```diff
@@ -96,6 +96,75 @@ Please refer to our :ref:`Choosing appropriate compute resources page <Choosing-
...
+* Did you request too much time? (e.g on Bessemer, more than 168 hrs.)
```
Comment (jkwmoore) [inline FAQs.rst:110]:
```suggestion
* Did you request too much time? (e.g for ShARC, more than 96 hours or on Bessemer, more than 168 hrs.) 
```

**Comment by jkwmoore** [comment]
Added two suggestions but otherwise happy to merge. Let me know when you do as I need to update a coworker on TD.

**Comment by Carldkennedy** [comment]
> Added two suggestions but otherwise happy to merge. Let me know when you do as I need to update a coworker on TD.

Thanks @jkwmoore. Done

**Pull request merged**